CHANGELOG.md

```diff
@@ -8,7 +8,12 @@ Remember to align the itemized text with the first line of an item within a list
 PLEASE REMEMBER TO CHANGE THE '..main' WITH AN ACTUAL TAG in GITHUB LINK.
 -->
 
-## jax 0.3.22
+## jax 0.3.22 (Oct 5, 2022)
```

Comment:
We only update the date when we actually make a release, sorry for the confusion. So please remove this date since we're not actually releasing.